ansible/roles/openregister_config/templates/openregister-config.yaml.j2

```diff
@@ -66,8 +66,9 @@ server:
       port: {% if paas %}${PORT}{% else %}8080{% endif %}
   registerDefaultExceptionMappers: false
   requestLog:
+    type: classic
     appenders:
-      - type: access-logstash-console
```

Comment:
Ah yes. I had forgotten that we were still using config from S3 for PaaS! Will update.